packages/data-connect/src/api/DataConnect.ts

```diff
@@ -111,10 +111,8 @@ export class DataConnect {
     }
   }
   // @internal
-  _useGeneratedSdk(): void {
-    if (!this._isUsingGeneratedSdk) {
-      this._isUsingGeneratedSdk = true;
-    }
+  _setCallerSdkType(callerSdkType: CallerSdkType): void {
```

Comment:
Can we keep the last function and then call the `_setCallerSdkType` under the hood? Just so that we aren't removing an existing function and can deprecate it later.